### PR TITLE
Update RU: add 2025 substituted holidays

### PR DESCRIPTION
**Pull request by Alexeyzhu**
## Proposed change

Add 2025 Russian Substituted Holidays.

## Type of change

- [ ] New country/market holidays support (thank you!)
- [x] Supported country/market holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency update (version deprecation/upgrade)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] I've run `make pre-commit`, it didn't generate any changes
- [x] I've run `make test`, all tests passed locally

[contributing-guidelines]: https://github.com/vacanza/python-holidays/blob/dev/CONTRIBUTING.rst
[docs]: https://github.com/vacanza/python-holidays/tree/dev/docs/source

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -137,3 +137,4 @@ Victor Luna
 Victor Miti
 Ville Skyttä
 Youhei Sakurai
+Zhuchkov Aleksei
```
Comment (arkid15r) [inline AUTHORS:140]: Please use first name last name order.

**Review by arkid15r — COMMENTED** [review]
Looks clean, just one comment regarding AUTHORS file format:

File: AUTHORS (hunk 1 above)
Comment (Alexeyzhu) [reply AUTHORS]: Updated file

File: AUTHORS [hunk 2]
```diff
@@ -6,6 +6,7 @@ Aart Goossens
 Abdelkhalek Boukli Hacene
 Akos Furton
 Alejandro Antunes
+Aleksei Zhuchko
```
Comment (arkid15r) [inline AUTHORS:9]: It looks like a typo.

**Review by arkid15r — APPROVED** [review]
@Alexeyzhu thanks for the contribution!

**Pull request merged**